simple-wp-helpdesk/includes/helpers.php

```
@@ -369,6 +387,169 @@ function swh_get_string_comment_meta( int $comment_id, string $key ): string {
+		}
+	}
+
+	if ( ! $assigned ) {
+		$default  = get_option( 'swh_default_assignee' );
+		$assigned = is_scalar( $default ) ? (int) $default : 0;
+	}
+
+	if ( $assigned > 0 ) {
+		update_post_meta( $ticket_id, '_ticket_assigned_to', $assigned );
+	}
+}
```

PHPMD (2.15.0)

[warning] The function swh_apply_assignment_rules() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)